Assets/HoloToolkit/Input/Scripts/InputSources/GesturesInput.cs

```diff
@@ -58,14 +67,19 @@ private void Awake()
                                                                     GestureSettings.NavigationY |
                                                                     GestureSettings.NavigationZ);
             }
-            navigationGestureRecognizer.StartCapturingGestures();
+
+            if (RecognizerStart == RecognizerStartBehavior.AutoStart)
+            {
+                gestureRecognizer.StartCapturingGestures();
+                navigationGestureRecognizer.StartCapturingGestures();
+            }
```

Comment:
It's just because all the ifs in StartGestureRecognizer are useless in here and it's how the both KeywordManager and SpeechInputSource have it.